src/components/Question.tsx

```diff
@@ -126,17 +127,34 @@ const StyledQuestion = styled.div`
       }
     }
 
+    // hide the radio button in all cases
```

Comment:
We defined a `visually-hidden` class for this kind of thing [in assignments](https://github.com/openstax/assignments/blob/main/packages/frontend/src/index.css#L34). You might want to do that here.